app/models/shared_edit_revision.rb

```diff
@@ -213,3 +213,23 @@ def self.revise!(post_id:, user_id:, client_id:, revision:, version:)
 # t.integer :user_id, null: false
 # t.integer :version, null: false
 # t.integer :post_revision
+
```

Comment:
we'll have to remove the old format annotations too

Reply:
I think the `rake annotate:clean:plugins` should handle it, right?

Reply:
Actually, this seems to be added by human hands.

Reply:
I searched for `# t.integer` in all-the and this is the only instance so I think can just remove this manually and we should be good

Reply:
https://github.com/discourse/discourse-shared-edits/commit/5fd7719f97ea5ebad493ca70a10d854e4c0a8328#diff-8a0728417277bb770c352f1f6566b7fb2455a61b0dd593d105f863edad3c9b69